common/pipeline_storage/src/main/scala/uk/ac/wellcome/pipeline_storage/PipelineStorageStream.scala

```diff
@@ -23,8 +24,9 @@ class PipelineStorageStream[In, Out, MsgDestination](
   messageStream: SQSStream[In],
   indexer: Indexer[Out],
   messageSender: MessageSender[MsgDestination])(
-  val config: PipelineStorageConfig)(implicit ec: ExecutionContext)
-    extends Logging {
+  val config: PipelineStorageConfig)(implicit val ec: ExecutionContext)
```

Comment:
Why has `ec` become a `val`?

Reply:
I'm not 100% sure.. After moving out the broadcastAndMerge thing it complained that it didn't have access to the ExecutionContext 🤷‍♀️

Reply:
Does the `logging` trait expect this perhaps?

Reply:
I don't think it's the logging, it shouldn't need the ExecutionContext. IIRC, in case classes any param in the class is automatically converted to a val so it's available to any method in the class. In normal classes, they're not so, unless you declare them val, they're only available when instantiating the class (I think). What I don't understand really, is why it ws working before..